compile/core.rb

```diff
@@ -127,6 +127,9 @@ def compile(file, caller_frame = 1)
       compiled = input.result(ctx)
       ctx.local_variable_set(:_erbout, content) if has_erbout # restore code
       compiled
+    rescue StandardError
```

Comment:
💯

Reply:
Does this output the original file name or the one being compiled?

(If I call `compile('templates/ansible/async.erb')` inside of `templates/ansible/resource.erb`, which filename will be outputted? 'async.erb' right?

Reply:
Yep. Previously, we got errors about the enclosing file (`resource.erb`), this adds errors for the inner file.